frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -60,20 +65,35 @@ const EditRecipeForm = ({ history }) => {
     setSelectedTagsWithId(tagsWithIdObject);
   }, [selectedTags]);
 
+  const validate = () => {
+    if (!title) {
+      setTitleError('Title cannot be blank');
+    } if (!description) {
+      setDescriptionError('Description cannot be blank');
+    } if (ingredients.length <= 1) {
+      setIngredientError('Ingredients cannot be blank');
+    } if (directions.length <= 1) {
+      setDirectionsError('Directions cannot be blank');
+    }
+  };
+
   const handleSubmit = () => {
     const parsedDuration = parseInt(duration, 10);
     const parsedServings = parseInt(servings, 10);
-
-    createRecipe({
-      title,
-      description,
-      difficulty,
-      duration: parsedDuration,
-      servings: parsedServings,
-      tags: selectedTagsWithId,
-      directions,
-      ingredients,
-    }).then(response => history.push(`/recipe/${response.data.id}`));
+    if (!title || !description || ingredients.length <= 1 || directions.length <= 1) {
+      validate();
```

Comment:
Personally, I'd be inclined to have `validate` return true or false based on whether an error was generated or not. Then `if (validate()) { createRecipe...` would be cleaner, and we wouldn't be checking the same if conditions twice.

But if @H12 is cool with the changes, I'm down to not block it.

Reply:
I agree with that feedback.